cap-tempfile/src/lib.rs

```diff
@@ -101,6 +101,15 @@ impl TempDir {
         Err(Self::already_exists())
     }
 
+    /// Make this directory persistent.
+    ///
+    /// This corresponds to [`tempfile::TempDir::into_path`].
```

Comment:
Could you add ", except that it returns a `Dir` instead of a `PathBuf`" to this comment?